native/cocos/platform/qnx/QNXPlatform.cpp

```diff
@@ -190,13 +72,12 @@ int32_t QnxPlatform::loop() {
     while (!_quit) {
         curTime         = getCurrentMillSecond();
 		desiredInterval = static_cast<long>(1000.0 / getFps());
-		
-        // pollEvent();
+        pollEvent();
         actualInterval = curTime - lastTime;
         if (actualInterval >= desiredInterval) {
             lastTime = getCurrentMillSecond();
             runTask();
-            //SDL_GL_SwapWindow(_handle);
+            //_sdl->swapWindow();
```

Comment:
Why to comment `_sdl` relevant  code?

Reply:
SDL has been adapted to QNX, but the adaptation is relatively simple, so I commented it out. In the future, if qnx is better supported, it can be opened up